core/src/main/resources/config.yml

```diff
@@ -5,9 +5,6 @@ general:
     enable-stat-collection: true
     enable-volatile-features: true
     enable-new-spell-handling: false
-    str-cast-usage: |
```

Comment:
Where is the equivalent functionality or has there already been an investigation to see if this was unused?

Reply:
As already mentioned, there is an automated [Help API](https://github.com/aikar/commands/wiki/Command-Help) (enabled with [risks](https://github.com/aikar/commands/wiki/Unstable-API)) which handles help based on the [Description annotation](https://github.com/JasperLorelai/MagicSpells/blob/4.0/core/src/main/java/com/nisovin/magicspells/MagicCommand.java#L191).

How this issue is resolved depends on your input in the [private dev channel](https://canary.discordapp.com/channels/335237931633606656/418982700985352192/748427019775377449). I won't resolve this until that input is given.